contracts/interfaces/IArrakisV2Factory.sol

```diff
@@ -9,10 +9,21 @@ interface IArrakisV2Factory {
 
     event InitFactory(address owner);
 
+    function initialize(address _owner_) external;
```

Comment:
why we need to put initialize function into the interface, cannot be called.

Reply:
yeah ok good point